content/blog/2023-05-16-comparison-of-ethereum-classic-and-ethereum-monetary-policies/index.md

```diff
@@ -0,0 +1,146 @@
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Hl1uKGW-40I" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+---
+
+In this post we will compare the Ethereum Classic (ETC) and Ethereum (ETH) monetary policies.
+
+When Ethereum forked away from ETC in 2016, the Ethereum Classic core developer team decided to establish a monetary policy as similar as possible as that of Bitcoin. The change occurred in December of 2017 and since then it has remained and will remain the same forever.
+
+Since genesis, Ethereum has had 5 monetary policies and they will likely keep changing them forever.
+
+In the table below we compare both blockchains’ monetary policies and in the following sections we will explain each feature in the table.
+
+![ETC and ETH monetary policies compared.](./etc-eth-table.png)
+
+## Ethereum’s Tendency to Centralization and Mega-Validators
+
+Before making the comparisons, it is important to note, as will be seen in the next sections, that Ethereum is a system that will tend to centralization, therefore censorship and arbitrary manipulation, not only because proof of stake does not impose any cost to changing the ledger, there is no objective focal point for exit and entry, and staking has very efficient economies of scale, but also because its monetary policy pays the staking industry, which will be dominated by very few players, enormous amounts of money per year, but at the same time burns coins from regular users who pay fees.
+
+This means that the staking industrial complex will perpetually get richer, creating mega-validators, while the rest of the economy will be constantly starved of money to pay for fees.
+
+While there will alway be some coin to pay for fees, there is a giant constant wealth transfer from the economy to the staking cartel.
+
+## Digital Gold
+
+ETC is digital gold and ETH is not.
+
+ETC is digital gold because it is a proof of work blockchain and the purpose of that consensus mechanism is to make it as costly to create ETC as it is costly to produce gold in the real world.
+
+The way it works is that miners in the blockchain create blocks of transactions and to stamp them cryptographically they need to do a lot of computational work, which uses a lot of electricity. Then, when the blocks are accepted by the rest of the network, the miners are credited newly issued coins by the algorithm.
+
+The above system guarantees that ETC will be sound money as gold was in the old days, hence the term “digital gold”.
+
+ETH is not digital gold because it is a proof of stake blockchain and validators do not have to do any work to produce blocks or earn rewards. They just receive transactions, package them into batches they call blocks to steal some lure from proof of work blockchains, and they vote on them to affirm their validity.
```

Comment:
Line 45: Wording correction
```suggestion
ETH is not digital gold because it is a proof of stake blockchain and validators do not have to do any work to produce blocks or earn rewards. They just receive transactions, package them into batches they call blocks to steal some of the allure of proof of work blockchains, and then vote on them to affirm their validity.
```